python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -543,6 +543,14 @@ def aten〇bmm(self: List[int], mat2: List[int]) -> List[int]:
     assert self[2] == mat2[1], "mismatching contracting dimension"
     return [self[0], self[1], mat2[2]]
 
+def aten〇baddbmm(self: List[int], batch1: List[int], batch2: List[int], beta: float = 1, alpha: float = 1) -> List[int]:
+    assert len(self) == 3, "baddbmm only supports 3D tensors"
```

Comment:
This is not right. `baddbmm` only expects the input tensor to be broadcastable to the result of the batch matmul of `batch1` and `batch2`. For example:

```python
In [1]: import torch

In [2]: a = torch.tensor(23)

In [3]: b = torch.rand((2, 2, 2))

In [4]: torch.baddbmm(a, b, b)
Out[4]: 
tensor([[[23.8636, 23.8087],
         [23.6450, 23.7825]],

        [[23.1581, 23.2126],
         [23.2608, 24.0102]]])
```

Can you also add a test that makes sure the broadcastability of `input` works correctly?